.env

```diff
@@ -1,2 +1,5 @@
 SERVER_PORT=8080
+DB_USERNAME=ccd
```

Comment:
what's this for?

Reply:
that's needed by shared ccd docker database